packages/logo/src/Logo.php

```diff
@@ -12,36 +12,36 @@
  */
 class Logo {
 	/**
-	 * Absolute URL of the Jetpack logo.
+	 * Jetpack logo as SVG shapes.
 	 *
 	 * @var string
 	 */
-	private $url;
+	private $logo;
 
 	/**
-	 * Constructor.
-	 * You can optionally pass a URL to override the default one.
-	 *
-	 * @param string $url New URL of the Jetpack logo.
+	 * Constructor. Initializes the logo property with a string describing the Jetpack logo.
+	 * The Jetpack logo SVG string includes CSS classes to stylize it:
+	 * - jetpack-logo: the wrapper <svg> tag.
+	 * - jetpack-logo__icon-circle: the circle of the Jetpack mark.
+	 * - jetpack-logo__icon-triangle: two shapes that correspond to each triangle in the Jetpack mark.
+	 * - jetpack-logo__icon-text: the Jetpack lettering.
 	 */
-	public function __construct( $url = '' ) {
-		if ( ! $url ) {
-			$url = plugins_url( 'assets/logo.svg', __DIR__ );
-		}
-
-		$this->url = $url;
+	public function __construct() {
+		$this->logo = '
+			<svg xmlns="http://www.w3.org/2000/svg" height="32" class="jetpack-logo" viewBox="0 0 118 32">
+				<path class="jetpack-logo__icon-circle" d="M16,0C7.2,0,0,7.2,0,16s7.2,16,16,16s16-7.2,16-16S24.8,0,16,0z" fill="#00be28" />
```

Comment:
I wonder it's better to include a stylesheet here and get rid of the inline `fill` attributes. What do you think? I'm fine if you decide to go YAGNI with it 😉

Reply:
I think it's fine because it's not a lot and we save a bunch of extra code.